### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by danintel**
* Upgrade answers to current 1.1 release (Bumper)
* Add Sawset error answer
* Mention log4rs for Debugging Rust
* Add answer on remote client access to REST API
* Add Hyperledger Sawtooth Explainer video
* Add Hyperledger Sawtooth Security videos
* Shorten YouTube URLs to https://youtu.be/
* Add detail about TCP port encryption
* Add State Pruning to glossary

Signed-off-by: danintel <daniel.anderson@intel.com>

### CODE REVIEW

**Review by dcmiddle — APPROVED** [review]

File: generator/source/faq/client.rst [hunk 1]
```diff
@@ -94,6 +94,11 @@ Use the ``delete_state`` in the SDK to delete a specific state variable.
 The data will remain in previously-created blocks (which are immutable),
 but will not be in the current blockchain state.
 
+How can a Sawtooth client access a validator on another machine?
+------------------------------
```
Comment (Sutrannu) [inline generator/source/faq/client.rst]: Title underlines should match the title character length.

File: generator/source/faq/rest.rst [hunk 2]
```diff
@@ -116,6 +116,10 @@ Back pressure is a flow-control technique to help prevent DoS attacks.
 It results in a ``Status.QUEUE_FULL`` client batch submit response or a 429 "Too Many Requests" REST API error.
 If the validator is overwhelmed it will stop accepting new batches until it can handle more work. The number of batches that validator can accept is based on a multiplier,  QUEUE_MULTIPLIER (currently 10, formerly 2), times a rolling average of the number of published batches.
 
+What does this error mean from sawset: ``There is no resource with the identifier``?
+-----------------------------------
```
Comment (Sutrannu) [inline generator/source/faq/rest.rst]: Title underlines should match the title character length.

Comment (danintel) [reply generator/source/faq/rest.rst]: Fixed and updated PR.

File: generator/source/faq/glossary.rst [hunk 3]
```diff
@@ -209,6 +209,8 @@ Solidity
     A contract-oriented programming language used to implement smart contracts. Compiles into Ethereum VM code and is supported by Seth
 Stale block
     A block proposed to be at the head of a blockchain, but lost to a competing block that became the head as decided by the consensus algorithm
+State Pruning
+    Removing uneeded older state roots from Merkle-Radix global state database.  See https://github.com/hyperledger/sawtooth-rfcs/pull/8
```
Comment (TomBarnes) [inline generator/source/faq/glossary.rst]: spelling: unneeded

Comment (danintel) [reply generator/source/faq/glossary.rst]: Fixed.

**Review by vaporos — APPROVED** [review]

**Comment by danintel** [comment]
Forced push to see if the CI checks work now.

**Comment by rberg2** [comment]
Hello. We recently fixed the whitelist check in the master branch. I restarted the build without that check since you are definitely an approved builder. Re-basing against master would pick up the fix too.

**Comment by danintel** [comment]
Thanks--it passes now and I merged it.  I tried a force push yesterday and it didn't work--maybe it was a little soon.

**Pull request merged**